bndtools.core/src/bndtools/views/repository/RepositoriesView.java

```diff
@@ -443,6 +460,31 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
         }.schedule();
     }
 
+    private void configureOfflineAction() {
+        Workspace workspace = Central.getWorkspaceIfPresent();
```

Comment:
I believe we need a 'Central.onInit()' style callback here.  this will block since this method calls getWorkspace will will download the internet if someone has a lot of jars in their repo.